packages/web/src/common/store/social/users/sagas.ts

```diff
@@ -48,9 +48,19 @@ export function* followUser(
   }
 
   const users = yield* select(getUsers, { ids: [action.userId, accountId] })
-  const followedUser = users[action.userId]
+  let followedUser = users[action.userId]
   const currentUser = users[accountId]
 
+  if (!followedUser) {
```

Comment:
do we need this check if we are assuming the forked fetchDefaultUsers will run beforehand?

Reply:
technically no, but this is more to future proof this kind of thing in the future for different flows